### PR TITLE
media-fonts/source-code-pro: add 2.042

### PR DESCRIPTION
**Pull request by mm1ke**
simple version bump for `media-fonts/source-code-pro`
I've also removed the blocker on `media-fonts/source-pro` as the package got removed almost 2 years ago.
 
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: media-fonts/source-code-pro/source-code-pro-2.042.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+REALV="2.042R-u/1.062R-i/1.026R-vf"
+
+DESCRIPTION="Monospaced font family for user interface and coding environments"
+HOMEPAGE="https://adobe-fonts.github.io/source-code-pro/"
+SRC_URI="https://github.com/adobe-fonts/source-code-pro/archive/${REALV}.tar.gz -> source-code-pro-${PV}.tar.gz"
+S="${WORKDIR}/${PN}-${REALV//\//-}"
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm arm64 ~ia64 ~loong ppc ppc64 ~riscv sparc x86 ~x64-macos"
```
Comment (tastytea) [inline media-fonts/source-code-pro/source-code-pro-2.042.ebuild:17]: please drop the KEYWORDS to testing

**Comment by tastytea** [comment]
thank you, must have slipped through the cracks 😅

File: media-fonts/source-code-pro/source-code-pro-2.042.ebuild (hunk 1 above)
Comment (mm1ke) [reply media-fonts/source-code-pro/source-code-pro-2.042.ebuild]: ah yeah.. forgot about that.. done :)

Comment (tastytea) [reply media-fonts/source-code-pro/source-code-pro-2.042.ebuild]: thanks! LGTM

**Pull request closed without merging**